Project/Views/UserProfile/_Story.cshtml

```diff
@@ -0,0 +1,40 @@
+﻿@model Project.ViewModels.Story.StoryVM
+<div class="card-body">
+    <div class="row">
+        <div class="col-8"><h3>Add Story</h3></div>
+    </div>
+        @using (Html.BeginForm("AddStory", "UserProfile", new { }, FormMethod.Post, new { @class = "form-horizontal", id = "addstory-form", role = "form" }))
+        {
+            @Html.AntiForgeryToken()
+
+            @Html.HiddenFor(x => x.Id)
```

Comment:
If this is an add only form, the Id is not needed. It will be automatically generated on insert.
Will this form be used for edits also?

Reply:
That's what I initially thought. But I don't think we can use it.